src/copy_tree.rs

```diff
@@ -13,11 +12,13 @@ use crate::*;
 pub struct CopyOptions {
     pub print_filenames: bool,
     pub measure_first: bool,
+    pub restore_only: String,
 }
 
 pub const COPY_DEFAULT: CopyOptions = CopyOptions {
     print_filenames: false,
     measure_first: false,
+    restore_only: String::new(),
```

Comment:
Just as general Rust style, I'd probably use `Option<String>` to represent that it can be unset, rather than a blank string.

Reply:
`Option<String>` has some limitations applying operations in Rust. I think I should convert to `Option<&str>` 
If you have a better solution, let me know

Reply:
If you use `Option<&str>` you'll need something else to hold the bytes for the lifetime of the `CopyOptions`, which could be messy. Or is the issue that `COPY_DEFAULT` is `const`? That seems like a mistake, rather than just making it `CopyOptions::default()`. I'll try to update that.

Reply:
That's fixed in b7134c1 so you should be good to use a `String` here now. Although I guess it should actually be an `Option<Apath>`.